akka-actor-tests/src/test/scala/akka/actor/ForwardActorSpec.scala

```diff
@@ -29,7 +30,7 @@ object ForwardActorSpec {
 
 class ForwardActorSpec extends AkkaSpec {
   import ForwardActorSpec._
-  implicit val ec = system.dispatcher
+  implicit val ec: ExecutionContextExecutor = system.dispatcher
```

Comment:
is this needed also if it was `private implicit val`?

if changing this manually I would prrobably use `: ExecutionContext`

Reply:
> is this needed also if it was `private implicit val`?

Yes.

> if changing this manually I would prrobably use `: ExecutionContext`

Change by `ExplicitResultTypes` scalafix rule